app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,67 @@
+class MoviesController < ApplicationController
+  def index
+    @category   = params[:category] || 'title'
+    @sort_order = params[:sort_order] || 'asc'
+    @search     = params[:search]
+    @active     = 'movies'
+    if @category || @search
+      @movie = Library.movies_search_sort(@search, @category, @sort_order)
```

Comment:
`@category` will always be true. Thus this will always be run.

Can clean it up by just always running this.